app/api_topologies.go

```diff
@@ -21,6 +21,14 @@ var (
 	topologyRegistry = &registry{
 		items: map[string]APITopologyDesc{},
 	}
+	pseudoFilter = APITopologyOptionGroup{
+		ID:      "pseudo",
+		Default: "show",
+		Options: []APITopologyOption{
+			{"show", "Show pseudo nodes", nil, false},
+			{"hide", "Hide pseudo nodes", render.IsNotPseudo, true},
```

Comment:
Agree its a shitty term for the UI.

Its _uncontained_ on the containers view, _unmanaged_ on the k8s views.  And also the internet nodes.